python/pyrogue/_Root.py

```diff
@@ -379,14 +385,14 @@ def _sendYamlFrame(self,yml):
         frame.write(b,0)
         self._sendFrame(frame)
 
-    def _streamYaml(self,modes=['RW','RO']):
+    def _streamYaml(self,modes=['RW','RO','WO'],saveEnFilt=False):
```

Comment:
I dont' think 'WO' should be included here

Reply:
Before it was not. "WO" variables are part of the configuration and should be included in the stream dump since they represent the current config.

Reply:
Plus this flag will allow the user to specifically set the variable to not be included if it should not.